dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelf.scala

```diff
@@ -500,6 +490,75 @@ case class BinaryOutcomeDLCWithSelf(
     }
   }
 
+  def executeJusticeDLC(
+      dlcSetup: SetupDLC,
+      timedOutCET: Transaction,
+      local: Boolean): Future[DLCOutcome] = {
+    val justiceOutput = timedOutCET.outputs.head
```

Comment:
can't we just use a `.find()` here instead of hard coding it? I guess this could probably be applied every where else in the file too where we use `.head` and `.last`?